cyhy_report/stakeholders/stakeholders.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```

Comment:
I know this is cribbed but we shouldn't be doing this anymore as the script in `bin/` negates the need.

```suggestion
```